setup/quickstart/halyard-local/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: single
+title:  "Halyard on your local machine Quickstart"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+In this quickstart, you will learn the basics of [Halyard](/setup/install/halyard/), Spinnaker's tool for managing your Spinnaker instance.
+
+NOTE: This Quickstart guide assumes you are running Ubuntu 14.04 LTS, but the process should be similar for other unix operating systems.
+
+## Overview
+
+In our scenario, we want to create a Spinnaker instance and set it up as follows:
+
+* The Spinnaker instance is itself running on your local machine
+* Spinnaker is downloaded from github, and built locally.
+* We use redis as our persistance store
+
+## Part 0: Prerequisites
+
+Ensure that the following are installed on your system
+
+* git
+* curl
+* yarn ([guide](https://yarnpkg.com/lang/en/docs/install/))
+
+Fork all of the microservices listed here: [Spinnaker Microservices](https://www.spinnaker.io/reference/architecture/#spinnaker-microservices) on github.
+
+Follow the following guides to setup ssh access to your github.com account from your local machine.
```

Comment:
I would add this as a separate header, since it might get lost under all the different prereqs. I'd probably format this as:

```
### Install local dependencies

### Fork all Spinnaker repos 

(maybe include a screenshot here of the 'fork' button on the github repo's page)

### Setup SSH keys

### Connect to Redis

Either 1. open a tunnel on 6379, or 2. run a local instance
```

Reply:
Done.